addons/sms/models/sms_sms.py

```diff
@@ -166,6 +166,18 @@ def _postprocess_iap_sent_sms(self, iap_results, failure_reason=None, delete_all
                         'failure_type': self.IAP_TO_SMS_STATE[state] if state != 'success' else False,
                         'failure_reason': failure_reason if failure_reason else False,
                     })
+                if state in ('success', 'insufficient_credit'):
+                    is_credit_error = False
+                    if state == 'insufficient_credit':
+                        notif_title = _('Not enough credits for SMS')
+                        is_credit_error = True
+                    else:
+                        notif_title = _('SMS is successfully sent')
+                    self.env['iap.account']._send_iap_notification(
```

Comment:
Most SMS are sent asynchronously. I don't think we should send notifications on bus in that case. This should be controlled by a parameter that is Falsy by default and enabled on demand. Purpose is to avoid unnecessary queries and bus polling, as we already have too many of them.